.github/workflows/ci.yml

```diff
@@ -72,3 +72,53 @@ jobs:
             cd build && cmake -DBUILD_BPF=On -DUSE_BUNDLED_DEPS=False ../
             KERNELDIR=/lib/modules/$(ls /lib/modules)/build make -j4
             make run-unit-tests
+
+  build-and-test-modern-bpf-x86:
+    name: build-and-test-modern-bpf-x86 🏗️🧪
+    runs-on: ubuntu-22.04
+    steps:
+
+      - name: Checkout Libs ⤵️
+        uses: actions/checkout@v3
+
+      - name: Install deps ⛓️
+        run: |
+          sudo apt update 
+          sudo apt install -y --no-install-recommends ca-certificates cmake build-essential clang-14 git pkg-config autoconf automake libtool libelf-dev
+
+      - name: Build scap-open and check no verifier issues 👀
+        run: |
+          mkdir -p build
+          cd build && cmake -DUSE_BUNDLED_DEPS=ON -DUSE_MODERN_BPF=ON -DBUILD_MODERN_BPF_TEST=ON -DMODERN_BPF_DEBUG_MODE=ON -DBUILD_LIBSCAP_GVISOR=OFF ../
+          make scap-open
+          sudo ./libscap/examples/01-open/scap-open --modern_bpf --num_events 0
```

Comment:
Not sure here if we want to keep the `build` step separated from the `run` step :thinking: Maybe it is better to have them separated... WDYT?

Reply:
I think it's fine having them as separate jobs. As I understand it, jobs are meant to group together actions taken by CI in order for us humans to have an easier time trying to understand what's going on. They all run in the same VM too, so there wouldn't be any overhead if you run everything in the same job or do something like:
- build scap-open
- build bpf-tests
- run scap-open
- run bpf-tests